build-go.sh

```diff
@@ -9,8 +9,9 @@ export GOPATH
 # Always use go modules
 export GO111MODULE=on
 
-DIST_PATH=${DIST_PATH:-/ipns/dist.ipfs.io}
-DIST_PATH=$(ipfs resolve "$DIST_PATH")
+# Content path to use when looking for pre-existing release data
+DIST_ROOT=${DIST_ROOT:-/ipns/dist.ipfs.io}
+DIST_ROOT=$(ipfs resolve "$DIST_ROOT")
```

Comment:
nit: Seems odd in one script to overwrite a variable like this.  Maybe do as one line?

Reply:
Simplified in 9acff7e :+1: